CameraInspector/Frame.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <memory>
+
+namespace cv { class Mat; }
+
+namespace CameraInspector
+{
+
+class Frame
+{
+public:
+	Frame();
+	Frame(int width, int height, void* data);
+	
+	void Construct(int width, int height, void* data);
+
+	int GetCols() const;
+	int GetRows() const;
+	void* GetData() const;
+
+private:
+	std::shared_ptr<cv::Mat> frame_;
```

Comment:
I don't like such name. He associates me with variable of the Frame class.
Maybe mat_ or cv_mat_impl_ will better choise...